### PR TITLE
Merge pull request #6545 from tkdrg/runtimes-killed-santa

### PR DESCRIPTION
**Pull request by Gribletastic**
Fixes christmas event runtime

### CODE REVIEW

File: code/modules/events/holiday/xmas.dm [hunk 1]
```diff
@@ -99,8 +99,8 @@
 			var/response = alert(M, "Santa is coming to town! Do you want to be santa?", "Ho ho ho!", "Yes", "No")
 			if(response == "Yes" && M && M.client && M.stat == DEAD && !santa)
 				santa = new /mob/living/carbon/human(pick(blobstart))
-				M.mind.transfer_to(santa)
 				santa.key = M.key
+				qdel(M)
```
Comment (Jalleo) [inline code/modules/events/holiday/xmas.dm]: Qdel dosent work on mobs i believe @mrperson should be able to confirm

Comment (MrPerson) [inline code/modules/events/holiday/xmas.dm]: qdel(mob) calls del(mob) anyway, but also calls ghostize() and some other cleanup procs.

Comment (Jalleo) [inline code/modules/events/holiday/xmas.dm]: Ah so it was sorted good to know MrP i only remember how it was said to work when first added.

**Comment by phil235** [comment]
@Gribletastic still working on this?

**Comment by Razharas** [comment]
Ask maintainer to reopen this if you decide to finish it

**Pull request closed without merging**